### DISCUSSION TITLE
Froze on Reverb G2 but worked well using Quest 2 with Airlink

### DISCUSSION THREAD

**Discussion by Braxusx** [fholger/crysis_vrmod#9] [General]
The latest version hooks with both Reverb G2 and Quest 2 using Airlink, but as soon as the actual game starts while using Reverb G2, the game freezes. 

By the way, I can't thank you enough for making this VR mod. You know how to pick them. Crysis is freaking awesome.

If you would like a tester, I can help out in that regards. I'm on discord along side you in the half life channel, but I see there isn't a section yet for Crysis. If you start one and need anything let me know. I own a Reverb G2, Quest 2, and Rift S I can use to test the game with.

So far I've played a good bit of it with Quest 2 using Airlink with no big problems so far. It looks really nice!

Cheers!

**Comment by fholger** [comment]
If you have anti-aliasing turned on, try to disable it or turn it down. I believe it increases the internal resolution, and there is a limit where it starts to cause issues and crashes - you might be reaching that with the G2. (Or it might be something else entirely.)
